cmd/metricswatcher/main.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import (
+	"os"
+	"sync"
+
+	"github.com/spf13/cobra"
+
+	"github.com/mattermost/mattermost-load-test-ng/config"
+)
+
+var configuration *config.Configuration
+
+func main() {
+	rootCmd := &cobra.Command{
+		Use:    "metricswatcher",
+		RunE:   runMetricsWatcher,
+		PreRun: config.Setup,
+	}
+
+	persistentFlags := rootCmd.PersistentFlags()
+	persistentFlags.StringP("config", "c", "", "path to the configuration file to use")
+	persistentFlags.StringP("queries", "q", "", "path to the JSON file with Prometheus queries")
+	cobra.MarkFlagRequired(persistentFlags, "queries")
+
+	if err := rootCmd.Execute(); err != nil {
+		os.Exit(1)
+	}
+}
+
+func runMetricsWatcher(cmd *cobra.Command, args []string) error {
+	var err error
+	configuration, err = config.GetConfig()
+
+	jsonQueryFile, _ := cmd.Flags().GetString("queries")
+
+	if err != nil {
+		return err
+	}
+
+	var wg sync.WaitGroup
+	wg.Add(2)
+
+	go healthcheck()
+	go checkMetrics(jsonQueryFile)
+
+	wg.Wait()
```

Comment:
I removed the waitgroup and I'm using error channels instead.
I keep some calls to `os.Exit` during the Cobra setup process because loadtest was doing it, so I decided to keep it consistent.